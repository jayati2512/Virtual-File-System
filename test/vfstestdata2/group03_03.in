createvfs vfs 100
mountvfs vfs
makedir / dir1
addfile / file.txt /usr/include/stab.h
addfile / file1.txt /usr/include/elf.h
listfile /file1.txt /tmp/output_list_file.txt
removefile /file1.txt
exportfile /file1.txt /tmp/output_export_file.txt
unmountvfs vfs
