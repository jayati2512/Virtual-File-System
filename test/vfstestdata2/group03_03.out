createvfs_SUCCESS
mountvfs_SUCCESS
makedir_SUCCESS
addfile_SUCCESS
addfile_FAILURE 	FILE_TOO_LARGE
listfile_FAILURE SOURCE_FILE_PATH_NOT_FOUND
removefile_FAILURE CANNOT_FIND_SPECIFIED_FILE
exportfile_FAILURE CANNOT_FIND_SOURCEFILE
unmountvfs_SUCCESS 
